src/main/java/com/aws/greengrass/util/platforms/windows/WindowsPlatform.java

```diff
@@ -162,8 +162,9 @@ public void validateDefaultConfiguration(Map<String, Object> proposedDeviceConfi
 
             @Override
             public Optional<RunWith> generate(DeviceConfiguration deviceConfig, Topics config) {
-                // TODO set the actual user name
-                return Optional.of(RunWith.builder().user(System.getProperty("user.name")).build());
+                // TODO do proper checks like UnixRunWithGenerator
+                String user = Coerce.toString(config.find(RUN_WITH_NAMESPACE_TOPIC, POSIX_USER_KEY));
```

Comment:
we need a windows user field, right?

Reply:
Yes. Can't really just posixUser it's confusing